temporaltest/config.go

```diff
@@ -0,0 +1,170 @@
+// The MIT License
+//
+// Copyright (c) 2022 Temporal Technologies Inc.  All rights reserved.
+//
+// Copyright (c) 2020 Uber Technologies, Inc.
+//
+// Copyright (c) 2021 Datadog, Inc.
```

Comment:
Can you confirm whether we need DataDog to confirm CLA sign to ensure we own this code and its licensing rights? Copyright can still remain of course, but externally contributed code to the server we need ownership confirmation. I think it's ok same as Temporalite, but should just check.

Maybe just a quick check with @rylandg